_includes/trusted.html

```diff
@@ -0,0 +1,50 @@
+<style type="text/css" scoped>
+  .trusted-section {
+    color: #eaf1f7;
+    background-color: #2c3e50;
+    padding: 4em;
+  }
+
+  .trusted-container {
```

Comment:
In BEM you would call this `trusted-section__container` to denote the significant relationship

Reply:
Fixed